CODEOWNERS

```diff
@@ -0,0 +1,65 @@
+# https://help.github.com/en/articles/about-code-owners
+# Documentation: https://github.com/hmcts/sds-flux-config/blob/master/CODEOWNERS.md
```

Comment:
this file isn't in place, dunno if want to link to cnp-flux-config one or duplicate and change environment names?

Reply:
Yeah I can create one for sds-flux-config. Just copied from cnp-flux-config basically so it's another one of those that has been curated for that repo

Reply:
sure